### PR TITLE
MongoDB lesson plan 

### PR DESCRIPTION
**Pull request by amychan331**
@allardbrain

### CODE REVIEW

**Comment by allardbrain** [comment]
Please move the content under "Guided Practice" to "Activity 1" under "Independent Practice." Please put the current content in "Independent Practice" as "Activity 2"

**Comment by amychan331** [comment]
@allardbrain The contents's been moved into Independent Practice, and I cleaned up some typos.

File: mongo-db/mongo-db.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+### Mini Lesson
+
+[MongodDB (video walkthrough of slides)]()
+
+[MongoDB (slides)](https://docs.google.com/presentation/d/1BvO6PrSpulHVSDNOkMaDZM-V7McmheLgm0Lg2PFae7k/edit#slide=id.p)
+
+
+### Common Mistakes / Misconceptions
+
+- **"I already learned relational databases. I'll just stick with that. They both store database anyway. "** The structure of relational databases are set up for relational datas. Using a mismatch databse for non-relational data results in reduce scalability and added cost.
```
Comment (rcoh) [inline mongo-db/mongo-db.md]: typo: store data

**Comment by amychan331** [comment]
Typo fixed. Thanks @rcoh !

**Comment by allardbrain** [comment]
Many thanks, Amy!

**Pull request merged**